pkgs/desktops/gnome-2/platform/libgnomeprint/default.nix

```
@@ -9,10 +9,16 @@ stdenv.mkDerivation rec {
     sha256 = "1034ec8651051f84d2424e7a1da61c530422cc20ce5b2d9e107e1e46778d9691";
   };
 
-  patches = [ ./bug653388.patch ];
+  patches = [
+    ./bug653388.patch
+    (fetchpatch {
+      url = "https://raw.githubusercontent.com/pld-linux/libgnomeprint/master/bison3.patch";
```

This should be pinned in case the file is updates:

```suggestion
      url = "https://github.com/pld-linux/libgnomeprint/raw/54c0f9c3675b86c53f6d77a5bc526ce9ef0e38cd/bison3.patch";
```